.github/workflows/ci.yml

```
@@ -0,0 +1,117 @@
+        # Test startup time (should be under 100ms for optimized config)
+        nvim --headless --startuptime startup.log -c "qa"
+        startup_time=$(tail -1 startup.log | awk '{print $1}')
+        echo "Startup time: ${startup_time}ms"
+
+        # Warn if startup is slow (but don't fail)
+        if (( $(echo "$startup_time > 200" | bc -l) )); then
+          echo "⚠️  Startup time is slower than expected: ${startup_time}ms"
+        fi
+
+    - name: Plugin count check
+      run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:5:6: Double quote to prevent globbing and word splitting

(shellcheck)